cockroachcloud/export-metrics.md

```diff
@@ -0,0 +1,264 @@
+
+To enable metrics export for your {{ site.data.products.dedicated }} cluster to Datadog, you can either enable the Datadog integration in your {{ site.data.products.dedicated }} Cloud Console, or on the command line via the [Cloud API](cloud-api.html):
+
+- To enable metrics export to Datadog using the Cloud Console, follow the [Monitor {{ site.data.products.dedicated }} with Datadog](https://www.cockroachlabs.com/docs/cockroachcloud/monitoring-page.html#monitor-cockroachdb-dedicated-with-datadog) instructions.
+- To enable metrics export to Datadog using the [Cloud API](cloud-api.html), follow the steps below.
+
+1. Find your {{ site.data.products.dedicated }} cluster ID:
+
+	1. Visit the {{ site.data.products.db }} console [cluster page](https://cockroachlabs.cloud/clusters).
+	1. Click on the name of your cluster.
+	1. Find your cluster ID in the URL of the single cluster overview page: `https://cockroachlabs.cloud/cluster/{your_cluster_id}/overview`. It should resemble `f78b7feb-b6cf-4396-9d7f-494982d7d81e`.
+
+1. Determine your cluster's cloud provider account ID. This command uses the third-party JSON parsing tool [`jq`](https://stedolan.github.io/jq/download/) to isolate just the needed `account_id` field:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request GET \
+      --url https://cockroachlabs.cloud/api/v1/clusters/{your_cluster_id} \
+      --header 'Authorization: Bearer {secret_key}' | jq .account_id
+    ~~~
+
+    Where:
+    - `{your_cluster_id}` is the cluster ID of your {{ site.data.products.dedicated }} cluster as determined in step 1.
+    - `{secret_key}` is your {{ site.data.products.dedicated }} API key. See [API Access](console-access-management.html) for more details.
+
+1. Determine the [Datadog API key](https://docs.datadoghq.com/account_management/api-app-keys/) you'd like to use. If you don't already have one, follow the steps to [add a new Datadog API key](https://docs.datadoghq.com/account_management/api-app-keys/#add-an-api-key-or-client-token).
+
+1. Issue the following Cloud API command to enable metrics export for your {{ site.data.products.dedicated }} cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request POST \
+      --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/metricexport \
+      --header "Authorization: Bearer {secret_key}" \
+      --data '{"datadog":{"site": "{datadog_site}", "api_key": "{datadog_api_key}}"}}'
+    ~~~
+
+    Where:
+    - `{cluster_id}` is your {{ site.data.products.dedicated }} cluster ID as determined in step 1, resembling `f78b7feb-b6cf-4396-9d7f-494982d7d81e`.
+    - `{secret_key}` is your {{ site.data.products.dedicated }} API key. See [API Access](console-access-management.html) for instructions on generating this key.
+    - `{datadog_site}` is your Datadog [site](https://docs.datadoghq.com/getting_started/site/), like `US1`.
```

Comment:
nit: should we enumerate all the possible options here as they should appear in the API request? They are:

- `US1`
- `US3`
- `US5`
- `US1_GOV`
- `EU1`

Reply:
True, but the options I listed are *our own internal* representation of the various Datadog sites. If a user tries making a request with `US1-FED` it would cause an error because it would not be a recognized value in our system.

Reply:
Oh! But why not use the upstream list, if we only deviate for one entry? Guess: internal list already in use elsewhere by other engineers?

Well, in any case, I've added the valid sites a user can provide here, and removed the link to DD sites reference, since it would confuse ("but the linked resource says `US1-FED`" etc).

Thanks for the clarification!